.gitignore

```diff
@@ -365,5 +365,6 @@ $RECYCLE.BIN/
 
 # build directory
 build
+SKIP_PREFLIGHT_CHECK=true 
```

Comment:
I propose to undo this edit. There's no reqmt to skip the preflight checks.

Reply:
okok